From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: dianlujitao <dianlujitao@lineageos.org>
Date: Thu, 27 Dec 2018 14:58:45 +0800
Subject: [PATCH 033/226] Enable Binder Proxy Tracking by Uid only on eng
 builds

 * App would be killed once limit reached, this is really bad for users.

Change-Id: I226a7a34031c9e9d8bf6843c22d79e72203bf867
---
 packages/SystemUI/src/com/android/systemui/SystemUIService.java | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/packages/SystemUI/src/com/android/systemui/SystemUIService.java b/packages/SystemUI/src/com/android/systemui/SystemUIService.java
index 0c43bb08ac..19af3d18e6 100644
--- a/packages/SystemUI/src/com/android/systemui/SystemUIService.java
+++ b/packages/SystemUI/src/com/android/systemui/SystemUIService.java
@@ -99,7 +99,7 @@ public class SystemUIService extends Service {
             throw new RuntimeException();
         }
 
-        if (Build.IS_DEBUGGABLE) {
+        if (Build.IS_ENG) {
             // b/71353150 - looking for leaked binder proxies
             BinderInternal.nSetBinderProxyCountEnabled(true);
             BinderInternal.nSetBinderProxyCountWatermarks(
